src/app/containers/ArticleMain/index.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { Fragment } from 'react';
+import { string, shape } from 'prop-types';
+import { articleDataPropTypes } from '../../models/propTypes/article';
+import MetadataContainer from '../Metadata';
```

Comment:
This is the right place but still feels weird to me. We have the metadata container in `ArticleMain` but the snapshots for `ArticleMain` do not output the metadata values. Think this is something to discuss and understand better...